src/pages/EmployeeProfile/components/BioItem.jsx

```diff
@@ -0,0 +1,35 @@
+import "./style.css";
+import { Typography, Box } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import { PropTypes } from "prop-types";
+
+/**
+ *renders Bio component of employee profile
+ * @returns {JSX.Element} Biography item for employee profile page component
+ */
+
+export default function BioItem(props) {
```

Comment:
Would BiographyItem not a better name?

Reply:
No. "_the life story of a person written by someone else._" - users typically write a biography of themselves (an _autobiography_).

Hence I would be open to calling it "AutobiographyItem".